src/System.Drawing.Common/tests/SystemFontsTests.cs

```diff
@@ -85,7 +134,42 @@ public void GetFontByName_NoSuchName_ReturnsNull(string systemFontName)
             Assert.Null(SystemFonts.GetFontByName(systemFontName));
         }
 
-        [DllImport("kernel32.dll", SetLastError = true, CharSet = CharSet.Auto)]
-        public static extern int GetSystemDefaultLCID();
+        [DllImport("kernel32.dll", SetLastError = false, CharSet = CharSet.Auto)]
+        public static extern int GetUserDefaultLCID();
+
+        //Do not test DefaultFont and DialogFont, as we can't reliably determine from LCID
+        //https://github.com/dotnet/corefx/issues/35664#issuecomment-473556522
+        class SystemFontList
+        {
+            public SystemFontList(string c_it_m_mb_scFonts)
+            {
+                CaptionFont = c_it_m_mb_scFonts;
+                IconTitleFont = c_it_m_mb_scFonts;
+                MenuFont = c_it_m_mb_scFonts;
+                MessageBoxFont = c_it_m_mb_scFonts;
+                SmallCaptionFont = c_it_m_mb_scFonts;
+                StatusFont = c_it_m_mb_scFonts;
+            }
+
+            public string CaptionFont { get; set; }
+            public string IconTitleFont { get; set; }
+            public string MenuFont { get; set; }
+            public string MessageBoxFont { get; set; }
+            public string SmallCaptionFont { get; set; }
+            public string StatusFont { get; set; }
+
+            public IEnumerable<object[]> ToTestData()
+            {
+                return new []
+                {
+                new object[] {(Func<Font>)(() => SystemFonts.CaptionFont), "CaptionFont", CaptionFont},
```

Comment:
"CaptionFont" => nameof(CaptionFont) ? Same for other properties.

Reply:
If you think so, sure.